hal_st/middlewares/ble_middleware/GattClientSt.cpp

```diff
@@ -173,6 +242,47 @@ namespace hal
         connectionHandle = GattClientSt::invalidConnection;
     }
 
+    void GattClientSt::HandleAttReadResponse(evt_blecore_aci* vendorEvent)
+    {
+        auto attReadResponse = *reinterpret_cast<aci_att_read_resp_event_rp0*>(vendorEvent->data);
+
+        infra::ByteRange data(&attReadResponse.Attribute_Value[0], &attReadResponse.Attribute_Value[0] + attReadResponse.Event_Data_Length);
+
+        really_assert(attReadResponse.Connection_Handle == connectionHandle);
+
+        if (onResponse)
+            onResponse(data);
+    }
+
+    void GattClientSt::HandleGattIndicationEvent(evt_blecore_aci* vendorEvent)
+    {
+        auto gattIndicationEvent = *reinterpret_cast<aci_gatt_indication_event_rp0*>(vendorEvent->data);
+
```

Comment:
Newlines are useful to separate different blocks. But if everything is a separate block, then you might as well combine everything into one block and remove all newlines

Reply:
Fixed.